test.txt

```diff
@@ -0,0 +1,2 @@
+ewhfahwefheoifhw
```

Comment:
You can delete this file as this is just a test file at the top of your repository that doesn't need to be committed.

You can actually add commits to your PR even after you open it, it's functionally a "work in progress" until it's closed (without any merge) or merged into `master` (or whatever branch you want to merge into). This means that you can just commit a change to this and delete this file.